chore(android): Disable NDK integration through Gradle (#6432)
getsentry/sentry-docs: src/platforms/android/using-ndk.mdx

You can use Sentry's Android SDK without the Android Native Development Kit (NDK) by either:

- [Disabling the NDK](#disable-ndk-integration)

> - Using `sentry-android-core`, which doesn't contain the NDK and can be used separately, instead of `sentry-android` when adding the dependency. The minimal required API level for `sentry-android-core` is 14.
> 
> If you're using our [Gradle plugin](/platforms/android/gradle/), it will still pull the NDK integration in as part of the auto-installation feature. To disable, you can remove the `sentry-android-ndk` dependency from the app configurations in `app/build.gradle`:
> 

Reviewer: ```suggestion
- Using `sentry-android-core`, which doesn't contain the NDK and can be used separately instead of `sentry-android`, when adding the dependency. The minimal required API level for `sentry-android-core` is 14.

If you're using our [Gradle plugin](/platforms/android/gradle/), it'll still pull the NDK integration in as part of the auto-installation feature. To disable it, remove the `sentry-android-ndk` dependency from the app configurations in `app/build.gradle`:

```
